pydarn/plotting/axis.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2021 SuperDARN Canada, University of Saskatchewan
+# Author: Daniel Billett
+#
+# Modifications:
+#
+# Disclaimer:
+# pyDARN is under the LGPL v3 license found in the root directory LICENSE.md
+# Everyone is permitted to copy and distribute verbatim copies of this license
+# document, but changing it is not allowed.
+#
+# This version of the GNU Lesser General Public License incorporates the terms
+# and conditions of version 3 of the GNU General Public License,
+# supplemented by the additional permissions listed below.
+"""
+Code which generates axis objects for use in plotting functions
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+class Projections():
+    """
+        Methods for generating axis objects for use in plotting functions
+        Methods
+        -------
+        axis_polar
+        """
+
+    def __str__(self):
+        return "This class is static class that provides"\
+                " the following methods: \n"\
+                "   - axis_polar()\n"\
+
+    @classmethod
+    def axis_polar(cls, lowlat: int = 30, hem: str='N'):
```

Comment:
Can we use: https://github.com/SuperDARN/pydarn/blob/b58da297e685f92dacc2ca30771222f7becbad6f/pydarn/utils/superdarn_radars.py#L181

To keep it consistent and then we can call on the station id to get this information?